src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs

```diff
@@ -257,6 +257,96 @@ void Check(bool expected, string msg)
 			Check(true, "Assert after another double click while myCheckBox should stay checked");
 		}
 
+		private void VerifyTextBlockText(string text1, string text2, string text3)
+		{
+			var result = _app.Marked("result");
+			var resultTapped = _app.Marked("resultTapped");
+			var resultCommand = _app.Marked("resultCommand");
+
+			Assert.AreEqual(text1, result.GetDependencyPropertyValue("Text")?.ToString());
+			Assert.AreEqual(text2, resultTapped.GetDependencyPropertyValue("Text")?.ToString());
+			Assert.AreEqual(text3, resultCommand.GetDependencyPropertyValue("Text")?.ToString());
+		}
+
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Uno.UITest.Helpers.Queries.Platform.iOS, Uno.UITest.Helpers.Queries.Platform.Android)] // Wasm is disabled https://github.com/unoplatform/uno/issues/2664
+		public void ButtonNative_Button01_Validation()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.Buttons_Native");
+
+			// Verify text of all text blocks initially
+			VerifyTextBlockText("No value", "No value", "No command");
+
+			var button01 = _app.Marked("button01");
+
+			// Click on button01 and verify text and number of clicks in all text blocks
+			button01.Tap();
```

Comment:
`.FastTap()` instead.
```suggestion
			button01.FastTap();
```
> \+ other places...